src/uct/ib/base/ib_iface.c

```diff
@@ -1032,7 +1032,13 @@ ucs_status_t uct_ib_verbs_create_cq(uct_ib_iface_t *iface, uct_ib_dir_t dir,
     }
 
     if (!cq) {
```

Comment:
[minor] pls, replace by `cq == NULL` here and above.
we prefer this direct check with `NULL`/`0`.